progetti_list.json

```diff
@@ -1,14 +1,12 @@
 {
-  "Firefox Desktop": "https://www.mozilla.org/it/firefox/new/",
   "Common Voice": "https://voice.mozilla.org/it",
-  "IHR (Internet Health Report)": "https://internethealthreport.org/",
-  "Firefox Mobile": "https://www.mozilla.org/it/firefox/mobile/",
+  "Firefox Desktop": "https://www.mozilla.org/it/firefox/new/",
+  "Firefox Fire Tv": "https://www.amazon.it/Mozilla-Firefox-for-Fire-TV/dp/B078B5YMPD",
   "Firefox Focus": "https://support.mozilla.org/it/kb/firefox-focus-android",
```

Comment:
- Rimuovere "Firefox Focus", "Firefox ScreenshotGo", "Firefox Reality" (discontinued)
- Aggiornare il link di Common Voice -> "commonvoice.mozilla.org"
- Aggiungere "Mozilla Voice" -> "voice.mozilla.org"
- Rinominare "MDN (Mozilla Developer Newtork)" in "MDN Web Docs"